### PR TITLE
Edited comments now updated according to its ID

### PR DESCRIPTION
**Pull request by MaksymShokin**
@tonytlwu @squallstar 

## Issue
Fliplet/fliplet-studio#5085

## Description
The bug occurred because editing comments was updating the last comment. Now it updates comment by id.

## Screenshots/screencasts
editing comment demo

## Backward compatibility
This change is fully backward compatible.

### CODE REVIEW

File: js/layout-javascript/news-feed-code.js [hunk 1]
```diff
@@ -2393,6 +2393,7 @@ DynamicList.prototype.saveComment = function(entryId, commentId, value) {
       return instance.update({
         settings: commentData.data.settings
       }, {
+        id: commentId,
```
Comment (tonytlwu) [inline js/layout-javascript/news-feed-code.js]: The `where` clause  beneath would be unnecessary if we pass the `id` property.

File: js/layout-javascript/simple-list-code.js [hunk 2]
```diff
@@ -2347,6 +2347,7 @@ DynamicList.prototype.saveComment = function(entryId, commentId, value) {
       return instance.update({
         settings: commentData.data.settings
       }, {
+        id: commentId,
```
Comment (tonytlwu) [inline js/layout-javascript/simple-list-code.js]: The `where` clause  beneath would be unnecessary if we pass the `id` property.

File: js/layout-javascript/news-feed-code.js (hunk 1 above)
Comment (MaksymShokin) [reply js/layout-javascript/news-feed-code.js]: fixed

File: js/layout-javascript/simple-list-code.js (hunk 2 above)
Comment (MaksymShokin) [reply js/layout-javascript/simple-list-code.js]: fixed

**Pull request merged**